app/assets/javascripts/data-tables-init.coffee

```diff
@@ -30,7 +30,7 @@ $(document).on "page:change", ->
 
     options =
       responsive: true
-      order: [[0, "desc"]]
+      order: [[0, "asc"]]
```

Comment:
This changed the default setting for the data-table initializer.

Please use the `th.sort-asc` on the colum to sort by. see line 37 in this same file.